commands/motor/speedmodifiers/SetEnableableModifier.java

```diff
@@ -2,29 +2,34 @@
 
 
 import org.usfirst.frc4904.standard.subsystems.motor.speedmodifiers.EnableableModifier;
-import edu.wpi.first.wpilibj.command.Command;
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import edu.wpi.first.wpilibj2.command.CommandScheduler;
 
-public class SetEnableableModifier extends Command {
+public class SetEnableableModifier extends CommandBase {
```

Comment:
I don't think i have a good understanding of this, so probably just ignore me, but is there a reason that this should extend commandbase instead of implementing command?